nilearn/_utils/docs.py

```diff
@@ -438,7 +438,7 @@ def custom_function(vertices):
 
 # hemi
 docdict["hemi"] = """
-hemi : {"left", "right", "both"}, default="left"
+hemi : {"left", "right", "both"}, default="both"
```

Comment:
Sorry if I am missing something but I read somewhere that we are keeping the default "left" in this PR?

Reply:
Yes, I've changed in the code, but not doc thinking that finally it will be `both`. Do you think I should keep them consistent? If yes, I can change this to `left` as well.

Reply:
I've changed that finally.